packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -233,7 +233,11 @@ contract L2Genesis is Deployer {
         setOptimismMintableERC20Factory(); // 12
         setL1BlockNumber(); // 13
         setL2ERC721Bridge(_l1Dependencies.l1ERC721BridgeProxy); // 14
-        setL1Block(); // 15
+        if (cfg.useInterop()) {
+            setL1BlockInterop(); // 15
```

Comment:
I recommend we put the conditional logic inside of `setL1Block` - it will reduce loc diff